docs/cluster-overview.md

```diff
@@ -65,7 +65,7 @@ The system currently supports several cluster managers:
 * [Standalone](spark-standalone.html) -- a simple cluster manager included with Spark that makes it
   easy to set up a cluster.
 * [Apache Mesos](running-on-mesos.html) -- a general cluster manager that can also run Hadoop MapReduce
-  and service applications.
+  and service applications. (Deprecated)
```

Comment:
I wasn't sure all the places we should mark this as deprecated, so, just sprinkled it across key documentation pages.

The source code looks all private to Spark, so, no real point in marking it as deprecated I think.